web/app/js/components/TapEventTable.jsx

```diff
@@ -36,34 +36,31 @@ const genFilterOptionList = options => _.map(options,  (_v, k) => {
 });
 
 let tapColumns = (filterOptions, ResourceLink) => [
-  {
-    title: "ID",
-    dataIndex: "requestInit.http.requestInit.id.stream"
-  },
   {
     title: "Direction",
+    key: "direction",
     dataIndex: "base.proxyDirection",
+    width: "60px",
     filters: [
       { text: "Inbound", value: "INBOUND" },
       { text: "Outbound", value: "OUTBOUND" }
```

Comment:
I think we should change the filter text here as well, to match the TO/FROM language that's used in the table.

![image](https://user-images.githubusercontent.com/9226/45380044-8d0e4c80-b5b6-11e8-86c5-1dec50701ff1.png)

Something like this:

```
    filters: [
      { text: "FROM", value: "INBOUND" },
      { text: "TO", value: "OUTBOUND" }
    ],
```